docs/machine-learning/index.yml

```diff
@@ -1,69 +1,107 @@
-    - html: <a href="/dotnet/machine-learning/tutorials/object-detection-onnx">Detect objects in images with ONNX</a>
-    - html: <a href="/dotnet/machine-learning/tutorials/text-classification-tf">Classify sentiment of movie reviews using TensorFlow</a>
-  - type: paragraph
-    text: '<b>Use Model Builder</b>, a graphical Visual Studio extension:'
-  - type: list
-    style: unordered
-    items:
-    - html: <a href="/dotnet/machine-learning/tutorials/predict-prices-with-model-builder">Predict prices with Model Builder</a>
-    - html: <a href="/dotnet/machine-learning/tutorials/sentiment-analysis-model-builder">Build a Razor Pages web application to analyze sentiment</a>
-  - type: paragraph
-    text: '<b>Run the ML.NET CLI</b>:'
-  - type: list
-    style: unordered
-    items:
-    - html: <a href="/dotnet/machine-learning/tutorials/mlnet-cli">Analyze sentiment with the ML.NET CLI</a>
-- title: Samples
-  items:
-  - type: paragraph
-    text: See the <a href="https://github.com/dotnet/machinelearning-samples">samples repository</a> on GitHub for applications in C# and F#.
-- title: Reference
-  items:
-  - type: paragraph
-    text: '<b>The ML.NET API</b>'
-  - type: list
-    style: unordered
-    items:
-    - html: 'The <a href="/dotnet/api/?view=ml-dotnet">Release ML.NET API</a> contains components for data handling, algorithms for binary classification, multiclass classification, regression, anomaly detection, time series and forecasting, ranking, model saving and loading, ONNX and TensorFlow model handling, and much more!'
-    - html: 'The <a href="/dotnet/api/?view=ml-dotnet-preview">Preview ML.NET API</a> contains algorithms for recommendation tasks, and support for standard deep learning models.'
-  - type: paragraph
-    text: '<b>The AutoML API</b>'
-  - type: list
-    style: unordered
-    items:
-    - html: '<a href="/dotnet/api/index?view=automl-dotnet">Auto ML API</a>'
-  - type: paragraph
-    text: '<b>ML.NET CLI reference</b>'
-  - type: list
-    style: unordered
-    items:
-    - html: '<a href="/dotnet/machine-learning/reference/ml-net-cli-reference">ML.NET CLI command reference</a>'
+  ms.author: nakersha
+  ms.date: 09/20/2019
+
+
+# linkListType: architecture | concept | deploy | download | get-started | how-to-guide | learn | overview | quickstart | reference | tutorial | video | whats-new
+
+landingContent:
+
+  # Card (optional)
+  - title: About ML.NET
+    linkLists:
+      - linkListType: overview
+        links:
+          - text: What is ML.NET?
+            url: how-does-mldotnet-work.md
+      - linkListType: concept
+        links:
+          - text: Machine learning model types
+            url: resources/tasks.md
+          - text: How to choose an algorithm
+            url: how-to-choose-an-ml-net-algorithm.md
+          - text: Data transformations
+            url: resources/transforms.md
+
```

Comment:
```suggestion
- linkListType: video
  links:
    - text: Introduction to Machine Learning and ML.NET Channel 9 series
      url: https://channel9.msdn.com/Series/MLNET
    - text: Introduction to Machine Learning and ML.NET YouTube series
      url: https://www.youtube.com/playlist?list=PLdo4fOcmZ0oUDTvk5XMNues09FnuB_D0u
```

Reply:
@luisquintanilla: Good idea to add a video. It's out of scope for this task (just conversion to the new format), but it can be added after this merges.